### ISSUE TITLE
Third air in act 2, scene 4

### ISSUE THREAD

**Issue by fredrowland** [BeggarsOperaTeam/Beggars-Opera-TEI#18]
Right after the third air, "When I once lay with another man's life", in the FE Jenny Diver says, "These are the Tools of a Man of Honour. Cards and Dice are only fit for cowardly Cheats, who prey upon their Friends."

In the SE and TE, she says, "A Man of Courage should never put any Thing to the Risque, but his Life. These are the Tools of a Man of Honour. Cards and Dice are only fit for cowardly Cheats, who prey upon their Friends."

Confirm with Steve that we go with SE and TE here.

**Comment by SteveNewman1970** [comment]
Yes-as per Fuller and Lewis, this looks to have authorial warrant.

From: Fred Rowland ***@***.***>
Sent: Wednesday, December 1, 2021 1:52 PM
To: BeggarsOperaTeam/Beggars-Opera-TEI ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [External] [BeggarsOperaTeam/Beggars-Opera-TEI] Third air in act 2, scene 4 (Issue #18)


Right after the third air, "When I once lay with another man's life", in the FE Jenny Diver says, "These are the Tools of a Man of Honour. Cards and Dice are only fit for cowardly Cheats, who prey upon their Friends."

In the SE and TE, she says, "A Man of Courage should never put any Thing to the Risque, but his Life. These are the Tools of a Man of Honour. Cards and Dice are only fit for cowardly Cheats, who prey upon their Friends."

Confirm with Steve that we go with SE and TE here.

-
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FBeggarsOperaTeam%2FBeggars-Opera-TEI%2Fissues%2F18&data=04%7C01%7Csteven.newman%40temple.edu%7Ce0bf2a940846432969d808d9b4fb96d6%7C716e81efb52244738e3110bd02ccf6e5%7C0%7C0%7C637739814925014764%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=qhyyRPOWM%2Bt%2F98HSGb1FhwAeZQWVieUVWAeJDd2Xz5Q%3D&reserved=0>, or unsubscribe<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FADNEF3HDK6RXIDLFJBPP6MDUOZVDFANCNFSM5JFG4FLA&data=04%7C01%7Csteven.newman%40temple.edu%7Ce0bf2a940846432969d808d9b4fb96d6%7C716e81efb52244738e3110bd02ccf6e5%7C0%7C0%7C637739814925024723%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=delkT3KmsdDDuHQj9wYQOSTo0w5xRuswAZA6NSoPhdQ%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Csteven.newman%40temple.edu%7Ce0bf2a940846432969d808d9b4fb96d6%7C716e81efb52244738e3110bd02ccf6e5%7C0%7C0%7C637739814925024723%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=majg3t4HHyYZaCzI13nHvU3XynSTu7q5HWeFlMYoNrM%3D&reserved=0> or Android<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7Csteven.newman%40temple.edu%7Ce0bf2a940846432969d808d9b4fb96d6%7C716e81efb52244738e3110bd02ccf6e5%7C0%7C0%7C637739814925034674%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=p9VTH2Zgnq5sMZKR76mP1XB1i%2Fp4BHrXfkkawuKNwNI%3D&reserved=0>.
